src/Extension/FluentExtension.php

```diff
@@ -638,21 +638,26 @@ public function onAfterWrite(): void
     public function onAfterDuplicate($original, $doWrite, $relations): void
     {
         $owner = $this->getOwner();
-        $localisedTables = $this->owner->getLocalisedTables();
+        $localisedTables = $owner->getLocalisedTables();
+
         // Get the names of all has_one columns that will have new IDs
         $copyRelations = (array) $owner->config()->get('localised_copy');
         $hasOne = $owner->hasOne();
         $copyHasOneRelations = [];
+
         foreach ($copyRelations as $relationName) {
-            if (array_key_exists($relationName, $hasOne)) {
-                $copyHasOneRelations[] = $relationName . 'ID';
+            if (!array_key_exists($relationName, $hasOne)) {
+                continue;
             }
```

Comment:
We should probably be checking if the relation is in `cascade_duplicates` as well, which I didn't think of when doing the original PR.
Edit: Actually we should be checking against `$relations` - which defaults to `cascade_duplicates` but could be anything if called manually.